extensions/resteasy-reactive/quarkus-resteasy-reactive-links/deployment/src/test/java/io/quarkus/resteasy/reactive/links/deployment/AbstractHalLinksTest.java

```diff
@@ -0,0 +1,38 @@
+package io.quarkus.resteasy.reactive.links.deployment;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.jboss.resteasy.reactive.common.util.RestMediaType;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import io.restassured.RestAssured;
+import io.restassured.parsing.Parser;
+import io.restassured.response.Response;
+
+public abstract class AbstractHalLinksTest {
+    @BeforeAll
+    public static void setup() {
```

Comment:
Should this be restored?

Reply:
I'm not sure about what you mean here

Reply:
`RestAssured.defaultParser` is being set before all tests are run. It's previous value should probably be restored or we could potentially encounter hard to debug issues in other tests

Reply:
I've just configured that setting the defaultParser was not necessary either :/ 
Deleted from the resteasy classic links and the reactive links.